### PR TITLE
Adding Hiddify Multiplatform Client

### PR DESCRIPTION
**Pull request by hiddify-com**

### CODE REVIEW

**Comment by RPRX** [comment]
> A multi-platform proxy client based on [Sing-box](https://github.com/SagerNet/sing-box) universal proxy tool-chain.

https://t.me/projectXtls/236

**Sing-box 已经一年多没有跟进 Xray 的新协议们了，包括 REALITY 的协议、安全更新等，事实上已不再完全兼容 Xray 的众多协议**

作为客户端而言，目前 Clash.Meta 对 Xray 众多协议的兼容性更好

**Comment by hiddify-com** [comment]
I understand
This was the reason i have put it as other clients

By the way, we currently support tls fragment and hope to have more correlation with xray core in future.

**Comment by Fangliding** [comment]
The author of Singbox said that project violates Singbox's GPL license, and adding  such a repo in the README may not be appropriate((

**Comment by Morrigan-Ship** [comment]
Hiddify never violates Singbox's GPL license.
We always respect open source thinking and do not violate open source licenses such as GPL

**Comment by Fangliding** [comment]
> Hiddify never violates Singbox's GPL license. We always respect open source thinking and do not violate open source licenses such as GPL

It's libcore linked to sing-box and used an weird modified GPL license

https://www.gnu.org/licenses/gpl-faq.html#ModifyGPL
> Such a modified license is almost certainly [incompatible with the GNU GPL](https://www.gnu.org/licenses/gpl-faq.en.html#WhatIsCompatible)

**Comment by Morrigan-Ship** [comment]
What I noticed from Hidify is that they always respect the license
And their only limitation is to publish in the ios market, which has not been published yet, and then a series of limitations will be reduced

But they are from Permissions and Restrictions Under GNU GPL Version 3 Section 7

**Comment by mmmray** [comment]
I originally tried to ignore this issue but this conversation is really something else.

Non-commercial clause is obviously not compatible with GPL even with the exception mechanism you described, and clearly against the wishes of the original author, who has stated multiple times towards the Hiddify developers what he thinks of this licensing. Just ask them yourself instead of trying to [weasel](https://github.com/hiddify/hiddify-core/commit/84cb85a49b5811c9cb9ca64606e812ab69ccfae1#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1dR2) yourself out of this situation. It's also highly deceptive to claim Hiddify has always been "open source" and never violated the license. Hiddify-next is still licensed under CC-BY-SA, which restricts usage *more* than the average proprietary client.

Even if you don't care about the license issue, there is the deceptive marketing. Hiddify also claimed to "be recommended by Xray", and when they were called out on it, they ignored the comments. I had to [file a PR](https://github.com/hiddify/Hiddify-Manager/pull/4238) to get them to act many months later. When they did fix it, they filed this PR to add their client at the *top* of the list.

In short, I think Hiddify is not a fair player at the moment. I strongly feel that in its current state and with the way the authors address concerns, it shouldn't be added to the README.

**Comment by hiddify-com** [comment]
@mmmray we will review the licensing concerns soon. Thank you for your consideration

**Comment by Morrigan-Ship** [comment]
> we will review the licensing concerns soon. Thank you for your consideration

Any updates?...
